### ISSUE TITLE
Add new hydro plants to KR parser

### ISSUE THREAD

**Issue by madsnedergaard** [electricityMap/electricitymap-contrib#3476]
New hydro plants have been added to the mix and the parser for Korea is throwing the following warnings:

```
New hydro plant seen - Seomjingang - 8.0MW
New hydro plant seen - Euiam - 14.8MW
```

Would be nice to update the parser so it can include these, although the value is really small 😅 

See this PR for reference: https://github.com/tmrowco/electricitymap-contrib/pull/3257

**Comment by IV1T3** [comment]
Can't this issue be closed, since already in release v0.229.0 [15205a90c9675cf7faebb069402110b4b010d106](https://github.com/electricityMap/electricitymap-contrib/releases/tag/v0.229.0)?

**Comment by madsnedergaard** [comment]
Yes, good catch - thanks! 🥳
